berbix/berbix-react
# 1.0.0 upgrade

### What does this pr do?
- bumps version to 1.0.0
- removes `environment` param
- removes use of authorization code in `onComplete`
- removes `clientId, template, phone, email` params
- added `showCloseModalButton` param, which allows for differentiation between modal with and without close button

### How should this be tested?
- Use [this](https://github.com/berbix/onboarding-sample-integrations/pull/6) new sample app. See README of that app for instructions on how to do so
